.azure-pipelines/packages.yml

```diff
@@ -205,17 +159,14 @@ jobs:
       containerregistrytype: Container Registry
       dockerComposeCommand: run package.alpine
 
-  - task: PublishPipelineArtifact@0
-    displayName: publish artifacts
-    inputs:
-      artifactName: linux-alpine-packages
-      targetPath: deploy/linux
+  - publish: deploy/linux
+    artifact: linux-alpine-packages
 
 - job: s3_upload
-  condition: and(succeeded(), not(startsWith(variables['Build.SourceBranch'], 'refs/tags/')))
+  condition: and(succeeded(), eq(variables['push_to_s3'], 'true'))
```

Comment:
Will `push_to_s3` default to `true` for any automated build?

Reply:
That is configured in Azure Pipelines, yes. The idea was to allow building this pipeline for other branches for testing. For example, I was running it for this branch (`lpimentel/clean-up-pipelines-redux`) manually and didn't want to push to S3. When run manually from the UI, you can change the default values of variables.

Another solution could be to change the previous condition from `not(startsWith(..., 'refs/tags/'))` to `eq(..., 'master'))`, which seems simpler now that I think about it. Thoughts?

![image](https://user-images.githubusercontent.com/1851278/80411963-13b29300-889b-11ea-80d3-46f38bf3adae.png)

Reply:
Yeah the `eq(..., 'master')` would be simpler syntax. As long as we can manually override with the `push_to_s3` variable, I'm fine with it 😄

Reply:
Changes this in e9bdbec7. Now it will only push `master` to s3, and only if `push_to_s3=true` (which is the default, but we can override when running the pipeline manually).